src/views/Farms/components/SearchInput.tsx

```diff
@@ -0,0 +1,126 @@
+import React, { useState, useRef } from 'react'
+import { Input, SearchIcon, IconButton } from '@pancakeswap-libs/uikit'
+import styled, { css } from 'styled-components'
+
+const StyledInput = styled(Input)`
+  border-radius: 1rem;
+  padding-right: 5rem;
+  transition: 0.3s;
+  margin-left: auto;
+  ${({ theme }) => theme.mediaQueries.sm} {
+    width: 100% !important;
+    padding: 0 1rem !important;
+    opacity: 1 !important;
```

Comment:
Why is `!important` needed? Typically the only reason to use important is to override a 3rd party library.

(also, pixel values)

Reply:
we don't have any style linting set up, but definitely, `!important` is to avoid